HomeView.vue

```diff
@@ -0,0 +1,25 @@
+<script setup lang="ts">
```

Comment:
Why a new HomeView file as been create here ?
If it's a mistake can you delete it please